### PR TITLE
It5

### PR DESCRIPTION
**Pull request by annaroyer**
Everything should be pretty much good to go it5/it6. BUT what do you guys think about the db seed for items? The csv file is storing them as unit price so initially I divided that by 100 in the seed file before storing it in the database. But now I'm thinking that maybe we want to store price in unit price form? In which case we would have to change the price that users entered inside of our model before creating each new record. Thoughts?

We could tag the instructors if we aren't sure about this -

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Item < ActiveRecord::Base
+  validates_presence_of :title, :description, :price, :image, presence: true
```
Comment (lnchambers) [inline app/models/item.rb]: validates_presence_of already validates presence: true

File: db/seeds.rb [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+require './app/models/item'
 
 
 class Seeds
   CSV.foreach("./db/csv/merchants.csv", headers: true, header_converters: :symbol) do |row|
     Merchant.create!(row.to_hash)
   end
+
+  CSV.foreach("./db/csv/items.csv", headers: true, header_converters: :symbol) do |row|
+    price = (row[:unit_price].to_f / 100).round(2)
```
Comment (lnchambers) [inline db/seeds.rb]: Maybe we should consider using BigDecimal. Thoughts?

**Review by lnchambers — CHANGES_REQUESTED** [review]

**Comment by lnchambers** [comment]
Hey Ali, Ilana, we would like some feedback on our items pricing. We are currently converting the price to dollars while seeding, but we are thinking about using BigDecimal instead. Thoughts?

**Comment by AliSchlereth** [comment]
@lnchambers @annaroyer @garcia50 Great question! You'll want to store the data in the same format it comes in as. Changing the format would mutate the data and potentially introduce minute changes which could significantly impact calculations, etc down the road.

**Review by AliSchlereth — COMMENTED** [review]
@lnchambers @annaroyer @garcia50 Great question! You'll want to store the data in the same format it comes in as. Changing the format would mutate the data and potentially introduce minute changes which could significantly impact calculations, etc down the road.

**Review by lnchambers — APPROVED** [review]

**Pull request merged**